### PR TITLE
Fix Test_EquationsOfState CMake configuration

### PR DESCRIPTION
**Pull request by nilsvu**
The Test_EquationsOfState library was not built before.

### CODE REVIEW

**Review by carmaza — DISMISSED** [review]

**Review by kidder — CHANGES_REQUESTED** [review]
Please add the library to the `test-libs-pointwise-functions` target in `cmake/SetupBuildStageTargets.cmake`

**Comment by nilsvu** [comment]
Done @kidder

**Comment by kidder** [comment]
@nilsleiffischer the change I asked for isn't showing up on github.  Are you sure you committed it?

File: cmake/SetupBuildStageTargets.cmake [hunk 1]
```diff
@@ -83,6 +83,7 @@ add_dependencies(test-libs-pointwise-functions
   Test_GrMhdAnalyticData
   Test_BurgersSolutions
   Test_ElasticitySolutions
+  Test_EquationsOfState
```
Comment (nilsvu) [inline cmake/SetupBuildStageTargets.cmake:86]: @kidder is this the change you mean? It does show up for me..  so perhaps it's GitHub acting weird?

Comment (kidder) [reply cmake/SetupBuildStageTargets.cmake:86]: yes, I see it now...

**Review by kidder — APPROVED** [review]

**Comment by nilsvu** [comment]
@kidder I think this is ready to go. @carmaza approved last week and all that changed was adding the lib to a test target.

**Pull request merged**